src/api/raw.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{Result, ErrorKind};
 use failure::ResultExt;
 
-/// RawRawApi generation data
+/// RawApi generation data
```

Comment:
:sweat_smile:

Reply:
It happens to the best of us :stuck_out_tongue: